mergin/utils.py

```diff
@@ -201,6 +201,8 @@ def conflicted_copy_file_name(path, user, version):
     head, tail = os.path.split(os.path.normpath(path))
     ext = ''.join(Path(tail).suffixes)
     file_name = tail.replace(ext, '')
+    if ext.lower() in (".qgz", ".qgs"):
+        ext += '~'
```

Comment:
@alexbruy could you please add a short comment why we're doing this, so that when someone's looking at the code, it's clear? :-)

Reply:
Done